homeassistant/components/geniushub/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Genius Hub",
   "documentation": "https://www.home-assistant.io/components/geniushub",
   "requirements": [
-    "geniushub-client==0.4.15"
+    "geniushub-client==0.5.00"
```

Comment:
Version is `0.5.00` (is tagged in github as `v0.5.00`), but testing reveals it still installs version `0.5.0`, so no harm done.